pkg/capabilities/consensus/ocr3/aggregators/identical.go

```diff
@@ -36,18 +36,18 @@ type counter struct {
 
 var _ types.Aggregator = (*identicalAggregator)(nil)
 
-func (a *identicalAggregator) Aggregate(_ *types.AggregationOutcome, observations map[ocrcommon.OracleID][]values.Value, f int) (*types.AggregationOutcome, error) {
+func (a *identicalAggregator) Aggregate(lggr logger.Logger, _ *types.AggregationOutcome, observations map[ocrcommon.OracleID][]values.Value, f int) (*types.AggregationOutcome, error) {
```

Comment:
Passing a logger as method arguments is generally a code smell. Why is this necessary, when `identicalAggregator` already has an internal logger?

Reply:
@cedric-cordenier What is going on here? The linter is flagging the unused field now in @chudilka1's PR